addons/sale/tests/test_sale_order_discount.py

```diff
@@ -108,6 +108,19 @@ def test_percent_discount_above_100(self):
         with self.assertRaises(ValidationError):
             self.wizard.write({'discount_percentage': 1.1, 'discount_type': 'sol_discount'})
 
+    def test_discount_translation(self):        
+        self.env['res.lang']._activate_lang(('es_AR'))     
+        self.wizard.write({
+            'discount_percentage': 0.1,
+            'discount_type': 'so_discount',
+        })
+        self.wizard.sale_order_id.partner_id.lang='es_AR'
+        discount_product = self.wizard._get_discount_product()
+        discount_product.with_context(lang='es_AR').name='descuento'
+        self.wizard.action_apply_discount()
+        discount_line = self.sale_order.order_line[-1]
+        self.assertEqual(discount_line.name,"descuento : 10.00%")
```

Comment:
```suggestion
        self.assertEqual(discount_line.name,"descuento: 10.00%")
```
I guess this is common thing in French, but you should leave out the space in English (and by extension, Odoo), see 0c53d2813324d946456030a6a189eab271c5bc48